backend/contextProperties.js

```diff
@@ -1,32 +1,21 @@
 /**
  * Wrapper functions to set commonly used fields on an 'context' object that is managed over the scope of a request.
  * Hopefully reduces the liklihood of mis-typing property names.
- * Note that by convention the controller(s) and Middleware use the res.locals property as the request scoped context.
+ * Note that by convention the controller(s) and Middleware use the req.session property as the request scoped context.
  * From controllers down to clients, client interceptors etc the context object is called 'context'.
  */
 
 /* eslint no-param-reassign: 2 */
 const setTokens = (context, accessToken, refreshToken) => {
-  context.accessToken = accessToken
-  context.refreshToken = refreshToken
+  context.access_token = accessToken
```

Comment:
had to change this as the names of the variables changed between the context and the cookie, so need to use the cookie values to keep consistent with other apps